### DISCUSSION TITLE
native base 3.4.x incompatiable with react-native-web 0.18.x

### DISCUSSION THREAD

**Discussion by kamals1986** [GeekyAnts/NativeBase#5375] [General]
Hello,

Is there an ongoing thread to support react-native-web 0.18.x version as the new 0.18.x version of react-native-web added breaking changes in the way styles are created.

**Comment by kamals1986** [comment]
https://github.com/GeekyAnts/NativeBase/issues/5075#issuecomment-1179852818

**Comment by ankit-tailor** [comment]
Hey @kamals1986, It is fixed now. You can check out latest version. Let me know if you still face any issues.
